### DISCUSSION TITLE
Add set_color pins to Text Displays and other light sources

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#4761] [Ideas and suggestions]
As mentioned in title, the addition of this feature would allow more decorative/creative use for text displays, lamps, and the likes of which that don't have the set_color pin at the moment.

**Comment by Psyduckdeluxe** [comment]
I dont even need RGB pins to be happy just being able to turn something off, would be godsent!  But yea RGB would be a bonus.

**Comment by Vectricity** [comment]
Makes no sense why this must be asked for as a feature. The lack of set_color is a major hindrance to my ability to properly customize my submarine.  Please resolve this.

**Reply by Konfuz** [reply]
The Mod [VisualVarietyPack](https://steamcommunity.com/sharedfiles/filedetails/?id=2161488150) Contains exactly what you are asking for. 

Would be a great addition to vanilla though. 

![image](https://user-images.githubusercontent.com/11818151/129352455-0fca9420-6e8f-4a7e-b291-f838544ecd07.png)

**Comment by 3e849f2e5c** [comment]
This has now been implemented in the development build, probably won't be seeing the pin in the upcoming update but the one after that.
